### PR TITLE
Merge upstream/master

### PR DESCRIPTION
**Pull request by jackblanc**
## Changes

Merging frontend-scaffold. Includes GitHub Actions. 

## Verification Steps

Hope it builds.

### CODE REVIEW

**Review by rymaju — COMMENTED** [review]
LGTM code-wise, but its failing checks for some reason 😕

File: src/auth/ducks/thunks.ts [hunk 1]
```diff
@@ -10,6 +10,7 @@ export const login = (
   loginRequest: LoginRequest,
 ): UserAuthenticationThunkAction<void> => {
   return (dispatch, getState, { authClient, tokenService }): Promise<void> => {
+    // TODO: dispatch loading
```
Comment (connernilsen) [inline src/auth/ducks/thunks.ts:13]: Are these (and by these I mean the other TODO too) going to be done in another PR or something?

**Review by connernilsen — COMMENTED** [review]
I think it looks good. Is there anything specific you wanted me to take a look at? If not, I'll just approve it if you want me to.
